### PR TITLE
Add normalize max length for active input(#16680)

### PR DESCRIPTION
**Pull request by s1lver**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Tests pass?   | yes
| Fixed issues  | #16680

### CODE REVIEW

File: framework/helpers/BaseHtml.php [hunk 1]
```diff
@@ -1326,6 +1326,7 @@ public static function activeInput($type, $model, $attribute, $options = [])
         }
 
         static::setActivePlaceholder($model, $attribute, $options);
+        self::normalizeMaxLength($model, $attribute, $options);
```
Comment (rob006) [inline framework/helpers/BaseHtml.php]: Seems that `normalizeMaxLength()` is now called twice for `activeTextInput()` and `activePasswordInput()`.

Comment (s1lver) [reply framework/helpers/BaseHtml.php]: @rob006 Yes indeed. Then you need to remove the normalization from `activeTextInput()`

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Looks good. Need a line for CHANGELOG then merging.

**Review by rob006 — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**